style.css

```diff
@@ -316,3 +304,310 @@ footer {
   justify-content: space-evenly;
   margin-top: 60px;
 }
+
+.nav-list {
+  display: none;
+}
+
+@media screen and (min-width: 1024px) {
+  body {
+    background-image: none;
+    max-width: 1400px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  /* Hero-section */
+  .header {
+    justify-content: center;
+    border-bottom: 1px solid #dddfe6;
+    padding-top: 38px;
+    padding-bottom: 38px;
+  }
+
+  .nav-list {
+    display: block;
+    display: flex;
+    gap: 20px;
+  }
+
+  .nav-link {
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #1a2236;
+    font-family: var(--ff-normal);
+    text-decoration: none;
+  }
+
+  .hero-section {
+    padding-top: 0;
+    padding-inline-start: 90px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .decoration {
+    display: none;
+  }
+
+  .decoration-desktop {
+    display: inline;
+    margin-right: 0;
+    flex: 1;
+  }
+
+  .hero-section1 {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    margin-bottom: 100px;
+  }
```

Comment:
- [ ] Great job with your design 👍 . Kindly make sure your hero text is vertically centered as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/188288301-28bdb1f2-f9fe-4bb2-b6bf-0bf418eaed59.png)  |  ![image](https://user-images.githubusercontent.com/74814780/188288289-d15a5c00-d24e-4cea-a0de-e946860fc870.png)